src/test/app/PayChan_test.cpp

```diff
@@ -946,7 +946,7 @@ struct PayChan_test : public beast::unit_test::suite
 
         Env env(*this);
         env.fund(XRP(10000), alice);
-        for (auto const a : bobs)
+        for (auto const& a : bobs)
```

Comment:
Nit: Can you change `a` to `b` here? Same for line 993. Since the collection is called `bobs`, I think `b` is a better name. Also, on line 959, we iterate through `bobs` and name the element `b`.

Reply:
`a` for "account", but I don't mind changing it to `b`. Although that may take this out of the running for shortest patch - four characters!

Reply:
Ha! It's your call then. I don't mind either way.